libs/search-widget/package.json

```diff
@@ -1,4 +1,33 @@
 {
   "name": "@nuclia/widget",
-  "version": "0.0.1"
+  "version": "1.0.0",
+  "description": "Nuclia search widget",
```

Comment:
Maybe we should use the plural form as we provide several widgets, no? (search bar, search results and viewer are distinct widgets)